src/docs.md

```diff
@@ -359,10 +359,16 @@ button.onclick = () => cadenzaClient.downloadData('{embeddingTargetId}', 'excel'
 
 <small>API: [CadenzaClient#reload](./classes/CadenzaClient.html#reload)</small>
 
+To reload a worksheet:
 ```javascript
 cadenzaClient.reload();
 ```
 
+To invalidate caches and reload:
+```javascript
+cadenzaClient.reload(true);
```

Comment:
Shouldn't here the parameter _true_ be wrapped in a JSON object?
`cadenzaClient.reload({invalidateCaches: true})`

Reply:
Right, I changed that too, thanks :+1:

Reply:
:+1: